src/UdpServer.php

```diff
@@ -107,8 +116,8 @@ private function processData($data, &$chunks)
                 }
                 break;
             default:
-                // unknown message
-                throw new InvalidMessageException("Unknown message type.", $data);
+                // message is not encoded
+                $dataDecoded = $data;
```

Comment:
pockej, pockej.. u sure?

Reply:
zdá se, že to tak opravdu funguje

Reply:
no, kazdopadne, takovej error jsem v logach nenasel, tak se to dost mozna ani nedovime :)